seed/challenges/02-javascript-algorithms-and-data-structures/functional-programming.json

```diff
@@ -40,9 +40,9 @@
         "Functional programming is a style of programming where solutions are simple, isolated functions, without any side effects outside of the function scope.",
         "<code>INPUT -> PROCESS -> OUTPUT</code>",
         "Functional programming is about:",
-        "1) Isolated functions - there is no dependence on external state, which includes global variables that are subject to change",
+        "1) Isolated functions - there is no dependence on the state of the program, which includes global variables that are subject to change",
         "2) Pure functions - the same input always gives the same output",
-        "3) Functions have no side effects - they do not mutate (change) external states",
+        "3) Functions have no side effects - they do not mutate, or change, the state of the program outside the function",
         "<h4>Instructions</h4>",
         "The members of Free Code Camp happen to love tea.",
         "In the code editor, the <code>prepareTea</code> and <code>getTea</code> functions are already defined for you. Call the <code>getTea</code> function to get 40 cups of tea for the team, and store them in the <code>tea4TeamFCC</code> variable."
```

Comment:
@terakilobyte your suggestions are appreciated. Can you commit to going through this section and completely overhauling the challenges yourself?